en-GB.lproj/Main.strings

```diff
@@ -1,42 +1,42 @@
 
 /* Class = "UILabel"; text = "Export TFP0"; ObjectID = "0lM-oV-NGn"; */
-"0lM-oV-NGn.text" = "Export TFP0";
+"0lM-oV-NGn.text" = "Exporteren TFP0";
 
 /* Class = "UIButton"; normalTitle = "Clean Diagnostics Data"; ObjectID = "2He-sb-oZG"; */
-"2He-sb-oZG.normalTitle" = "Clean Diagnostics Data";
+"2He-sb-oZG.normalTitle" = "Schone diagnosegegevens";
 
 /* Class = "UINavigationItem"; title = "Settings"; ObjectID = "2mQ-1t-c11"; */
-"2mQ-1t-c11.title" = "Settings";
+"2mQ-1t-c11.title" = "Instellingen";
 
 /* Class = "UIButton"; normalTitle = "Done"; ObjectID = "36t-As-Rhs"; */
-"36t-As-Rhs.normalTitle" = "Done";
+"36t-As-Rhs.normalTitle" = "Klaar";
 
 /* Class = "UIButton"; normalTitle = "Done"; ObjectID = "380-mY-NVs"; */
-"380-mY-NVs.normalTitle" = "Done";
+"380-mY-NVs.normalTitle" = "Klaar";
 
 /* Class = "UIButton"; normalTitle = "@benjweaverdev (Development)"; ObjectID = "4tc-UD-CU7"; */
-"4tc-UD-CU7.normalTitle" = "@benjweaverdev (Development)";
+"4tc-UD-CU7.normalTitle" = "@benjweaverdev (Ontwikkelaar)";
 
 /* Class = "UILabel"; text = "Set Boot Nonce"; ObjectID = "5Ee-VR-cUR"; */
-"5Ee-VR-cUR.text" = "Set Boot Nonce";
+"5Ee-VR-cUR.text" = "Vestig Boot Nonce";
```

Comment:
"Stel Boot Nonce in" would be more fitting. I don't know how you came up with "Vestig". Did you use Google Translate?